client/jetpack-cloud/components/sidebar/index.tsx

```diff
@@ -8,24 +18,45 @@ import './style.scss';
 type Props = {
 	className?: string;
 };
-const Sidebar = ( { className }: Props ) => (
-	<nav className={ classNames( 'jetpack-cloud-sidebar', className ) }>
-		<header className="jetpack-cloud-sidebar__header">Header</header>
-		<div className="jetpack-cloud-sidebar__main">
-			<ul role="menu" className="jetpack-cloud-sidebar__navigation-list">
-				<li
-					className={ classNames(
-						'jetpack-cloud-sidebar__navigation-item',
-						'jetpack-cloud-sidebar__navigation-item--highlighted'
-					) }
-				>
-					Navigation items
-				</li>
-				<li className="jetpack-cloud-sidebar__navigation-item">Will go here</li>
-			</ul>
-		</div>
-		<div className="jetpack-cloud-sidebar__footer">Footer</div>
-	</nav>
-);
+const JetpackCloudSidebar = ( { className }: Props ) => {
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const jetpackAdminUrl = useSelector( ( state ) =>
+		siteId ? getJetpackAdminUrl( state, siteId ) : null
+	);
 
-export default Sidebar;
+	const translate = useTranslate();
+
+	return (
+		<Sidebar className={ classNames( 'jetpack-cloud-sidebar', className ) }>
+			<SidebarHeader>Header</SidebarHeader>
+
+			<SidebarMain>
+				<ul role="menu" className="jetpack-cloud-sidebar__navigation-list">
+					<li
+						className={ classNames(
+							'jetpack-cloud-sidebar__navigation-item',
+							'jetpack-cloud-sidebar__navigation-item--highlighted'
+						) }
+					>
+						Navigation items
+					</li>
+					<li className="jetpack-cloud-sidebar__navigation-item">Will go here</li>
+				</ul>
+			</SidebarMain>
+
+			<SidebarFooter>
+				{ jetpackAdminUrl && (
+					<SidebarItem
```

Comment:
This is not the final component we will use here. Ideally, we can reuse the `SidebarNavigatorMenuItem` component we will build later.